components/dada-execute/src/permission/my.rs

```diff
@@ -43,12 +43,14 @@ impl My {
 
     pub(super) fn check_read(&self, interpreter: &Interpreter) -> eyre::Result<()> {
         self.given.check_still_valid(interpreter)?;
-        Ok(self.tenant.cancel_tenant_if_exclusive(interpreter))
+        self.tenant.cancel_tenant_if_exclusive(interpreter);
+        Ok(())
```

Comment:
Is this in response to a clippy warning? This feels like a regression to me -- I set it up that way explicitly, because I wanted a compilation error if the return type of `cancel_tenant_if_exclusive` changes from `()` to something else. Admittedly, it looks a bit weird and I don't always do this very consistently.

Reply:
(Sometimes I think I'd prefer to have Rust warn if you ever don't consume a return value, full stop.)

Reply:
(No need to change anything here, just me grumbling.)

Reply:
![image](https://c.tenor.com/jYZmxGqvQKsAAAAC/old-man-yells-at-cloud-yelling.gif)

Reply:
It's this one: https://rust-lang.github.io/rust-clippy/master/index.html#unit_arg

```shell
error: passing a unit value to a function
  --> components/dada-execute/src/permission/my.rs:46:9
   |
46 |         Ok(self.tenant.cancel_tenant_if_exclusive(interpreter))
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: `-D clippy::unit-arg` implied by `-D warnings`
   = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#unit_arg
help: move the expression in front of the call and replace it with the unit literal `()`
   |
46 ~         self.tenant.cancel_tenant_if_exclusive(interpreter);
47 +         Ok(())
   |

error: passing a unit value to a function
  --> components/dada-execute/src/permission/my.rs:51:9
   |
51 |         Ok(self.tenant.cancel_tenant(interpreter))
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#unit_arg
help: move the expression in front of the call and replace it with the unit literal `()`
   |
51 ~         self.tenant.cancel_tenant(interpreter);
52 +         Ok(())
```

Reply:
OK. Not sure I agree with this particular one, but I'm going to let it ride. :)